### PR TITLE
doc add job posting

### PR DESCRIPTION
**Pull request by tacaswell**
- DOC: update new section to add job posting
- FIX: make show target look at right directory

### CODE REVIEW

**Comment by tacaswell** [comment]
🐑 I failed at using the gh command line tools and pushed to the wrong fork, sorry about that!

I do not think it is worth closing this and re-opening from my fork but I can if anyone wants me to for the sake of form.

File: docs/body.html [hunk 1]
```diff
@@ -132,15 +132,23 @@ <h1>Matplotlib: Visualization with Python</h1>
...
+            <h5 class="date">Feburary 17, 2022</h5>
+            <a href="https://discourse.matplotlib.org/t/research-software-engineer-roses-2020-position/22597">Matplotlib is hiring!</a>
             <p>
-              Highlights include <a href="https://matplotlib.org/devdocs/users/prev_whats_new/whats_new_3.5.0.html#subplot-mosaic-supports-simple-axes-sharing">shared axes in <span class="pre">subplot_mosaic</span></a>, post-colormaping <a href="https://matplotlib.org/devdocs/users/prev_whats_new/whats_new_3.5.0.html#image-interpolation-now-possible-at-rgba-stage"> resampling</a>, and support for <a href="https://matplotlib.org/devdocs/users/prev_whats_new/whats_new_3.5.0.html#gtk-backend">GTK4</a> and
-              <a href="https://matplotlib.org/devdocs/users/prev_whats_new/whats_new_3.5.0.html#qt-backends">Qt6</a>.
-</p><p>
-              See the <a href="https://matplotlib.org/devdocs/users/prev_whats_new/whats_new_3.5.0.html">release notes</a> and <a href="https://matplotlib.org/devdocs/api/prev_api_changes/api_changes_3.5.0.html">API changes</a> for details.
+              We are happy to post the 3-yeas Research Software Engineer position
```
Comment (greglucas) [inline docs/body.html:138]:
```suggestion
              We are happy to post the 3-year Research Software Engineer position
```

File: docs/body.html [hunk 2]
```diff
@@ -132,15 +132,23 @@ <h1>Matplotlib: Visualization with Python</h1>
...
+              funded by
+              the <a href="https://discourse.matplotlib.org/t/maplotlib-selected-for-nasa-roses-ostfl-2020/22303">NASA
+                ROSES-OSTFL 2020</a> grant.
+
+              See <a href="https://discourse.matplotlib.org/t/research-software-engineer-roses-2020-position/22597">the job description on discourse</a> and
+              submit as application
```
Comment (greglucas) [inline docs/body.html:144]:
```suggestion
              submit an application
```

**Review by greglucas — APPROVED** [review]
Minor typos

**Pull request merged**